### ISSUE TITLE
0.18.25 - no longer starting due to missing files

### ISSUE THREAD

**Issue by bplein** [factoriotools/factorio-docker#317]
I have a long running set of game servers using this container. They failed to come up with 0.18.25 image.

Here is the relevant error:
```
   0.884 Info PlayerData.cpp:68: Local player-data.json available, timestamp 1589925687
   0.884 Info PlayerData.cpp:75: Cloud player-data.json unavailable
   0.886 Factorio initialised
   0.887 Error CommandLineMultiplayer.cpp:372: Hosting multiplayer game failed: filesystem error: directory iterator cannot open directory: No such file or directory [/opt/factorio/saves]
   0.889 Info ServerMultiplayerManager.cpp:136: Quitting multiplayer connection.
   0.889 Info ServerMultiplayerManager.cpp:771: updateTick(4294967295) changing state from(Ready) to(Closed)
   0.923 Goodbye

```
Here is my docker-compose template:
```
version: '2.2'
services:
  factorio:
    image: "factoriotools/factorio:${LATEST}"
#    cpus: 2
#    cpuset: $MYCPU
    mem_limit: 12G
    mem_reservation: 4G
    restart: always
    network_mode: bridge
    ports:
     - "${GAME_PORT}:${GAME_PORT}/udp"
     - "${RCON_PORT}:${RCON_PORT}/tcp"
    environment:
      - PORT=${GAME_PORT}
      - RCON_PORT=${RCON_PORT}
      - UPDATE_MODS_ON_START=true
    container_name: "${CONTAINER_NAME}"
    volumes:
     - ./volume:/factorio
     - ../scenarios:/factorio/scenarios
     - ${PWD}/volume/config/config.ini:/opt/factorio/config/config.ini
    stdin_open: true
    tty: true
```

Did anything change about symlinks etc inside the image? My saves are locally under /volume/saves which in game is mounted to /factorio/saves and I think symlinked by the container image to /opt/factorio/saves.

Let me know. Thanks.

**Comment by SuperSandro2000** [comment]
Yeah, symlinks changed yesterday and I did not enough testing to catch this regression.

https://github.com/factoriotools/factorio-docker/commit/64a1f8d9bf42e5b4f11db199bc32b5287f22628e#diff-01ae538bcfc4d52e3d99609ada16b83b

**Comment by SuperSandro2000** [comment]
I think the problem is that the config is not changing. I'll do a PR and I would appreciate it if you would test it.

**Comment by bplein** [comment]
The problem is that the old saves and other directories aren’t under /opt anymore due to missing symlinks. A new container might be able to game save but they aren’t on the mounted volume.

**Comment by SuperSandro2000** [comment]
> The problem is that the old saves and other directories aren’t under /opt anymore due to missing symlinks.

I removed them in an attempt to fix #311.

> A new container might be able to game save but they aren’t on the mounted volume.

That's only true if you are mounting the config.ini. I did not consider this case and I am sorry for the inconvenience.

You should be able to change your config.ini with the values below and get it back working. I think I need to replace the static config with a sed scrip.t

```ini
[path]
write-data=/factorio
```

**Comment by bplein** [comment]
Changing the config.ini allows the server to come up. 

I'm happy to test a PR if I have the time when it's posted.

**Comment by bplein** [comment]
I looks like that fix may have broken 0.18.26, I've opened dup a new ticket for that. #321
